WORDS/main_Services/common_actions_service.py

```diff
@@ -19,38 +19,58 @@ def get_words_from_usr():
         get_words_from_usr()
 
 # Getting the possible password range from the user
+# def get_pass_range_from_user():
+#     try:
+#         pass_length = input('What is the range length of the password? example 6-12:')
+#         pass_length_range = pass_length.split('-')
+#         start = int(pass_length_range[0])
+#         end = int(pass_length_range[1])
+#         if(len(pass_length_range) == 2 and start < end):
+#             return start, end
+#         print(
+#             """
+#             !error!\n
+#             please follow given structure\n
+#             type two number as range numbers with '-' between them\n
+#             second number most be bigger than first number
+#             """
+#             )
+#         sleep(2)
+#         get_pass_range_from_user()
+#     except:
+#         print(
+#             """
+#             !error!\n
+#             please follow given structure\n
+#             type two number as range numbers with '-' between them\n
+#             second number most be bigger than first number\n
+#             dont type anything else
+#             """
+#             )
+#         sleep(2)
+#         # recall the function
+#         # تابع بازگشتی
+#         get_pass_range_from_user()
+
 def get_pass_range_from_user():
-    try:
-        pass_length = input('What is the range length of the password? example 6-12:')
-        pass_length_range = pass_length.split('-')
-        start = int(pass_length_range[0])
-        end = int(pass_length_range[1])
-        if(len(pass_length_range) == 2 and start < end):
+    while True:
+        try:
+            pass_length = input('What is the range length of the password? example 6-12:')
+            start, end = map(int, pass_length.split('-'))
+            if end <= start:
+                raise ValueError
             return start, end
-        print(
-            """
-            !error!\n
-            please follow given structure\n
-            type two number as range numbers with '-' between them\n
-            second number most be bigger than first number
-            """
+        except (ValueError, IndexError):
+            print(
+                """
+                !error!\n
+                please follow given structure\n
+                type two numbers as range numbers with '-' between them\n
+                Second number should be bigger than first number
+                """
             )
-        sleep(2)
-        get_pass_range_from_user()
-    except:
-        print(
-            """
-            !error!\n
-            please follow given structure\n
-            type two number as range numbers with '-' between them\n
-            second number most be bigger than first number\n
-            dont type anything else
-            """
-            )
-        sleep(2)
-        # recall the function
-        # تابع بازگشتی
-        get_pass_range_from_user()

```

Comment:
are you serious?!

Reply:
Because there was a problem with my commit inside vs code, I sent PR several times

Reply:
this is not a reason. fix your tools man